CIP-0067/registry.json

```diff
@@ -18,5 +18,5 @@
     "asset_name_label": 444,
     "class": "SFT",
     "description": "CIP-0068 - Datum Metadata Standard (444 sub standard)"
-  }
+  },
```

Comment:
Makes sense. thanks!

Reply:
Hmm. I thought trailing commas are not allowed in json. I know they are allowed in cbor diag, but I thought json was older and more strict.

Reply:
Note that before I made this change, there was no trailing comma.

Reply:
Actually @AndrewWestberg I mucked this up. 😖 The whole year or so I've been looking at CIP-0010 labels & now these, my mind has been turning it into a JS array rather than JSON.  This is probably because of daily work on the Developer Portal's portfolio arrays like `[showcase.js](https://github.com/cardano-foundation/developer-portal/blob/staging/src/data/showcases.js)` which would be a murder of merge conflicts without being able to enforce those trailing commas.  Just committed to reverse the change.